### PR TITLE
Avatar and button

### PR DESCRIPTION
**Pull request by purvasharmadev**
Added Avatar and Button Components

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,8 +74,32 @@ <h1>Badge</h1>
...
+    <hr>
+
+        <!-- Buttons -->
+        <h1>Buttons</h1>
+        <button class="btn">Default Button</button>
+        <button class="btn btn-primary">Primary Button</button>
+        <button class="btn btn-secondary">Secondary button</button>
+        <button class="btn btn-primary-outline">Primary Outline</button>
+        <button class="btn btn-secondary-outline">Secondary Outline</button>
+        <button class="btn btn-primary" ><a class="btn-link" href="#">link</a></button>
```
Comment (itsmebipul00) [inline index.html:89]: Here you can remove the button tag and design the a tag however you want

**Comment by itsmebipul00** [comment]
Also you can use rems in many more places

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+/* Icon */
+.btn-icon{
+    font-size: 1rem;
+    padding: 0.5rem 1.5rem;
+    color: var(--PRIMARY-COLOR);
+}
+
+/* Float */
```
Comment (iRohitGaur) [inline Components/Button/button.css:49]: Give consistent spacing

**Review by iRohitGaur — APPROVED** [review]
Good to merge

**Review by zoef-yt — APPROVED** [review]
I would suggest naming your colour variable like (--colour-primary) just a suggestion. Else looks good to merge.

File: Components/Avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/component.css">
+    <title>Avatar</title>
+</head>
+<body>
+    <h1>Avatar</h1>
+    <div>
+        <img src="/Assets/developer.png"  class="avatar avatar-sm" alt="" srcset="">
```
Comment (dcs-soni) [inline Components/Avatar/avatar.html]: You should provide the `alt` attribute in the image tag. Else looks great 👍

File: index.html [hunk 4]
```diff
@@ -40,6 +40,19 @@ <h1>Alerts</h1>
         Warning! Indicates a warning that might need attention.
     </div>
     <hr>
+    <h1>Avatar</h1>
+    <!-- Avatar Circle -->
+        <img src="/Assets/developer.png"  class="avatar avatar-sm" alt="" srcset="">
```
Comment (dcs-soni) [inline index.html:45]: Here also, the `alt` attribute is not provided. You can include if time persists.

File: index.html [hunk 5]
```diff
@@ -40,6 +40,19 @@ <h1>Alerts</h1>
...
+        <img src="/Assets/developer.png"  class="avatar avatar-md" alt="" srcset="">
+        <img src="/Assets/developer.png"  class="avatar avatar-lg" alt="" srcset="">
+
+        <br>
+        <!-- Avatar Square -->
+        <img src="/Assets/developer.png"  class="avatar-square avatar-sm" alt="" srcset="">
```
Comment (dcs-soni) [inline index.html:51]: Same, `alt` attribute.

**Review by dcs-soni — CHANGES_REQUESTED** [review]

File: Components/Button/button.css [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+
+/* Outline */
+.btn-primary-outline{
+    border: solid var(--PRIMARY-COLOR) 1px;
+    color: var(--PRIMARY-COLOR);
+}
+
+.btn-secondary-outline{
+    border: solid var(--SECONDARY-COLOR) 1px;
```
Comment (dcs-soni) [inline Components/Button/button.css]: The `border` is the shorthand property of `border-width`, `border-style`, `border-color` in respective order. To maintain consistency in all your CSS try to use the exact way for better readability. Here, it must be `border: 1px solid var(--PRIMARY-COLOR)`.

**Review by dcs-soni — CHANGES_REQUESTED** [review]

File: Components/Avatar/avatar.html (hunk 3 above)
Comment (purvasharmadev) [reply Components/Avatar/avatar.html]: Thankyou for pointing it out. I will correct it ASAP. Thankyou for your review.

File: Components/Button/button.css (hunk 6 above)
Comment (purvasharmadev) [reply Components/Button/button.css]: Thankyou for reviewing the code. The changes you mentioned will help me to write consistent code and readable code.

**Review by dcs-soni — APPROVED** [review]

**Review by dcs-soni — APPROVED** [review]

**Comment by purvasharmadev** [comment]
> I would suggest naming your colour variable like (--colour-primary) just a suggestion. Else looks good to merge.

Thankyou for reviewing and approving the code. Your suggestions are considerable.

File: index.html (hunk 1 above)
Comment (purvasharmadev) [reply index.html:89]: Thankyou for the code review , I will surely do the changes you suggested.

**Comment by purvasharmadev** [comment]
> Good to merge
Thankyou for reviewing and approving my code.

**Pull request merged**